<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Win_Login</class>
 <widget class="QMainWindow" name="Win_Login">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Win_Login</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="ag_background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix/res/bg_login.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ag_icon_username">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>260</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix/res/icon_user.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="ag_edt_username">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:4px; 
font-size:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>请输入账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ag_edt_password">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>270</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:4px; 
font-size:20px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>请输入密码</string>
    </property>
   </widget>
   <widget class="QLabel" name="ag_icon_password">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>340</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix/res/icon_password.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ag_icon_refind">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>340</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix/res/icon_wenhao.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ag_btn_login">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>434</y>
      <width>230</width>
      <height>40</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
background-color: #FF3B3B;
color: white;
border-radius:4px; 
}

QWidget:hover
{
background-color: #CC7B7B;
color: white;
}

QWidget:pressed
{
background-color: #892121;
color: white;
}</string>
    </property>
    <property name="text">
     <string>登陆</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ag_btn_close">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>40</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget
{
border-image: url(:/new/prefix/res/icon_x.png);
}

QWidget:hover
{
border-image: url(:/new/prefix/res/icon_x_hover.png);
}

QWidget:pressed
{
border-image: url(:/new/prefix/res/icon_x_press.png);
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QLabel" name="ag_hint">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>100</y>
      <width>400</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
